UC-Berkeley-Extension/global
# Main nav

## Asana Ticket(s)
- See ticket https://app.asana.com/0/0/1198894218701637/f

## Description
- Creating main navigation using Layout builder
- Creating haas nav, and limiting visibility via term conditions

## Testing instructions
- `ddev start`, `ddev ssh`, `composer install`
- `drush cim`, `drush cr`
- In theme root, run `npm start`
- Go into Menu, and create menu items for main nav (/admin/structure/menu/manage/main) and haas according to directions seen on https://docs.google.com/document/d/1mlO_wlFxPMV5SJGdBO65urVsUROOn1PmA4ivI_ZraQo/edit#. (Creating New Menu)
- Place the blocks as seen on ^^ doc.
- See that the navigation block shows on the header, and the drop down displays three column menu.
<img width="1328" alt="Screen Shot 2020-10-22 at 1 45 38 PM" src="https://user-images.githubusercontent.com/10488517/96916276-f129bd00-146c-11eb-9e33-e04e4b23455a.png">

- See that any page that is tagged with Haas taxonomy, it shows the Haas main menu.
<img width="798" alt="Screen Shot 2020-10-22 at 1 54 13 PM" src="https://user-images.githubusercontent.com/10488517/96917110-14a13780-146e-11eb-94aa-9dd5f4aed9c2.png">
